azure-pipelines.yml

```diff
@@ -13,6 +13,13 @@ parameters:
   - test
   - real
 
+schedules:
+- cron: 0 6 * * 0
```

Comment:
Thats a good idea to build the repo from time to time in order to make sure that the build still works